visualization/app/codeCharta/ui/legendPanel/legendPanel.component.html

```diff
@@ -3,6 +3,9 @@
 	ng-class="{ visible: $ctrl._viewModel.isLegendVisible, sideBarVisible: $ctrl._viewModel.isSideBarVisible }"
 >
 	<div ng-if="!$ctrl._viewModel.isDeltaState">
+		<div class="legend-block bold" ng-if="$ctrl._viewModel.colorMetric">
```

Comment:
is `ng-if` even required? The colorMetric should be set once the map is loaded.

Reply:
This is just to be sure, that (in case the colorMetric event has not already been triggered) it will not lead to a white margin in the legend. Do you agree to keep the ng-if?

Reply:
But if no file is loaded and therefore the color-metric is not set, we display a loading-gif that blurs the rest.
Can you provide a .gif where this results in some ugly behavior when you remove the `ng-if`?

Reply:
I agree that this can not happen at all and remove the ng-if. Thank you!